plugin/path_config_test.go

```diff
@@ -2,6 +2,8 @@ package gcpauth
 
 import (
 	"context"
+	"fmt"
+	"github.com/hashicorp/vault/sdk/helper/authmetadata"
```

Comment:
nit: import ordering

Reply:
Fixed the imports but I won't link the commit since I'm going to squash them all before I mark this ready as a PR.